SampleProject/Assets/Scripts/Title.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Title : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
[SA1500] If a statement spans multiple lines, the opening curly bracket must be placed on its own line.